h/models/organization.py

```diff
@@ -60,4 +54,4 @@ def validate_logo(self, key, logo):
         return logo
 
     def __repr__(self):
-        return '<Organization: %s>' % self.slug
+        return '<Organization: %s>' % self.pubid
```

Comment:
Testing in the shell I noted that `str(...)`-ing an Organization previously failed before the organization has been saved.